### ISSUE TITLE
[UI/UX] Missing alt text for Howlongtobeat and Libretro icons on platform page

### ISSUE THREAD

**Issue by NotaInutilis** [rommapp/romm#4040]
<img width="717" height="211" alt="Image" src="https://github.com/user-attachments/assets/e5aa0934-5d1f-4103-bf70-48b38308477b" />

On a platform page, metadata sources with a clickable link also have an alt text that shows up when hovering.

At the far right, Howlongtobeat and Libretro are only icons, with no text nor link, and their alt text does not show up when hovering. The inspector shows that both of these icons have a title and an alt, but the hover text does not appear on both Firefox and Edge.

An additional question regarding the missing links: it does not seem that Howlongtobeat has dedicated platform pages, but it appears libretro has per platform public repositories https://github.com/libretro/libretro-database#sources Why not link those here? Or is the link I found completely unrelated?

**Comment by Spinnich** [comment]
With HLTB, I think this would be easy enough beyond them breaking their API monthly. As for Libretro, pretty sure the stuff we get just pulls from here (https://thumbnails.libretro.com) which doesn't really have a "per game" page.
